src/modules/site-v2/base/views/tools/phenotype_database.py

```diff
@@ -228,3 +309,15 @@ def report(id):
 
     'JobStatus': JobStatus,
   })
+
+def convert_joined_query_tojson(q):
+  """
+    Convert PhenotypeMetadata query joined with PhenotypeDatabase to JSON
+  """
+  json_data = []
+  for trait in q:
```

Comment:
Not mandatory, but --

This JSON conversion seems like a pretty basic & useful operation, so if you want, you can actually move it to the `PhenotypeMetadata` class itself.  Might look something like:

```
def to_json(self):
    json_trait = super().to_json()  # Change to super to avoid infinite recursion
    phenotype_values = [ v.to_json() for v in self.phenotype_values ]
    json_trait['phenotype_values'] = phenotype_values
    return json_trait
```

You could then directly call it with `trait.to_json()` for each `trait` in your list.

Up to you whether this makes sense.  And if it does, whether it should be the default `to_json` behavior, or whether it should be a new method (like `to_json_with_values` or something).

Reply:
(Just realized -- you don't have to move the `super()` call to the start of the method.  I just did that when I was testing to make sure it worked.)